### ISSUE TITLE
GraphMap support in transform

### ISSUE THREAD

**Issue by dmyyy** [blitzarx1/egui_graphs#76]
I'm using GraphMap so this would be useful for me. Could make sense to add a Transformable trait, or even better `impl Into<Graph<N, E, Ty>> for GraphMap<N, E, Ty>`. Not sure I understand the use case for `to_input_graph_custom`.

**Comment by blitzarx1** [comment]
Good idea. I ll take a look into it on the upcoming week. If you wish you can file a PR as well.

You can check the usage of `to_input_graph_custom` in the examples. But I agree that variant with trait implementation would be more seamless and straightforward integration for users of the lib.

**Comment by blitzarx1** [comment]
Sorry for long response. I ve recently added `From` support for new public API type `Graph`. It support `petgraph::StableGraph` conversion to `Graph` type. #77 

In the following PRs I will also support `GraphMap` as we discussed here.

**Comment by blitzarx1** [comment]
Long time since my last visit here. @dmyyy  if you think you can take this one as well, please go for it. I don't feel like getting this done soon. I am focused on other issues now.

**Comment by dmyyy** [comment]
If I get something working I'll put out a pr for this one as well - not my focus currently but we'll see if I get to it
